csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -6,37 +6,33 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base class for sinkers for all possible segments of the sequence.</para>
+    /// <para>Представляет базовый класс для проходчиков по всем возможным сегментам последовательности.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase{T, Segment{T}}"/>
     public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segment<T>>
     {
         /// <summary>
-        /// <para>
-        /// Creates the segment using the specified elements.
-        /// </para>
-        /// <para></para>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="Segment"/> .</para>
```

Comment:
This is not a constructor, but a virtual method.
```suggestion
        /// <para>Создаёт новый экземпляр класса <see cref="Segment{T}"/> на основе последовательности <paramref name="elements"/>.</para>
```